packages/ripple-keypairs/src/Sha512.ts

```diff
@@ -1,39 +1,36 @@
-/* eslint-disable no-bitwise --
- * lots of bitwise operators necessary for this */
-import * as hashjs from 'hash.js'
-import BigNum = require('bn.js')
+import { sha512 } from '@xrplf/isomorphic/sha512'
+import { bytesToNumberBE } from '@noble/curves/abstract/utils'
+
+type Input = Uint8Array | number[] | string
```

Comment:
Can't this type be imported from `@xrplf/isomorphic`?

Reply:
It's not currently being exported, though perhaps it should ?